test/SourceKit/ExpressionType/filtered.swift

```diff
@@ -36,23 +36,23 @@ class Proto2Conformer: Proto2 {}
 func foo(_ c: Proto2Conformer) { _ = c }
 
 // BOTH: <ExpressionTypes>
-// BOTH: (126, 130): Clas
+// BOTH: (126, 130): Clazz
```

Comment:
The test failure output indicates that the renaming caused the output of this test to change in more ways than substitution of the `Clazz` for `Clas` can account for. My recommendation would be to drop the change in this file since `Clas` was already an intentional misspelling of `Class` and it's unlikely worth the effort of further updating this test to switch to a different spelling.